src/docker/shared.rs

```diff
@@ -647,6 +648,23 @@ impl MountFinder {
 
         path.to_path_buf()
     }
+
+    #[allow(unused_variables, clippy::needless_return)]
+    fn find_path(&self, path: &Path, host: bool) -> Result<String> {
+        #[cfg(target_os = "windows")]
+        {
+            // On Windows, we can not mount the directory name directly. Instead, we use wslpath to convert the path to a linux compatible path.
+            if host {
+                return Ok(path.to_utf8()?.to_string());
+            } else {
+                return path.as_wslpath();
+            }
+        }
+        #[cfg(not(target_os = "windows"))]
+        {
+            return Ok(self.find_mount_path(path).to_utf8()?.to_string());
```

Comment:
I think, just for future-proofing, we should make cross fail if `CONTAINER_IN_CONTAINER` is enabled and `os = windows`. I don't think that scenario will happen but it blocks it as we know it wont work

Reply:
I've added a check at the start of the docker command to fail if running on Windows and in docker:

https://github.com/cross-rs/cross/pull/904/files#diff-7f0024537bbf8484f3e68ec502bd9cff7805e1f6eaee0eb072647a14e1bdb36fR31-R37